canopeum_backend/canopeum_backend/models.py

```diff
@@ -156,6 +157,22 @@ class Site(models.Model):
     announcement = models.ForeignKey(Announcement, models.SET_NULL, blank=True, null=True)
     image = models.ForeignKey(Asset, models.SET_NULL, blank=True, null=True)
 
+    def get_plant_count(self):
+        site_species = Sitetreespecies.objects.filter(site=self)
+        return reduce(lambda x, y: x + y.quantity, site_species, 0)
```

Comment:
A reduce+add is just a sum ;)
```suggestion
        return sum(specie.quantity for specie in site_species)
```